src/Compiler/Facilities/BuildGraph.fsi

```diff
@@ -47,6 +48,8 @@ type NodeCodeBuilder =
     /// that a proper generic 'use' could be implemented but has not currently been necessary)
     member Using: CompilationGlobalsScope * (CompilationGlobalsScope -> NodeCode<'T>) -> NodeCode<'T>
 
+    member Using: IDisposable * (IDisposable -> NodeCode<'T>) -> NodeCode<'T>
```

Comment:
Comment please

Reply:
Done - please correct me if it's inaccurate.